### DISCUSSION TITLE
Run without encryption?

### DISCUSSION THREAD

**Discussion by sky9mike** [dashingsoft/pyarmor#1678] [Q&A]
I'd like to run try RFT mode without the encryption to review what it does. Is this possible?

**Comment by jondy** [comment]
Search `RFT` in the online documentation, you will find the solution.

**Comment by sky9mike** [comment]
When I use the following:

```
	pyarmor cfg trace_rft 1
	pyarmor cfg optimize 2
	pyarmor cfg mix.str:excludes "/.{2,}/"
	$(PYARMOR_SCRIPT) gen --mix-str --enable-rft $(ENTRY_MODULE);
```
The file in .pyarmor does not protect any strings. Also, I see """ comment blocks. I'd like those removed to to be able to show example of basic protection. Thanks.
